src/ui/Omnibox/Omnibox.ts

```diff
@@ -271,7 +272,13 @@ export class Omnibox extends Component {
      *
      * **Default:** `false`
      */
-    clearFiltersOnNewQuery: ComponentOptions.buildBooleanOption({ defaultValue: false })
+    clearFiltersOnNewQuery: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+    /**
+     * The amount of focus time (in milliseconds) required on a query suggestion before requesting a preview of its top results.
+     *
+     * **Default:** `200`
```

Comment:
You can remove **Default**: `200`; TypeDoc will automatically generate a tag from the buildNumberOption argument.

Reply:
Perfect, I'm removing that.